### ISSUE TITLE
support Trove quotas

### ISSUE THREAD

**Issue by aborrero** [gophercloud/gophercloud#3208]
Thanks for your hard work with this library, it is really appreciated.

I think, as of this writing, gophercloud is missing support for Openstack Trove quotas.

Please, add support for it.

See also:
* https://docs.openstack.org/api-ref/database/#quotas

**Comment by aborrero** [comment]
The mentioned patch only introduces the GET action on the endpoint.

I believe it misses the the PUT operation: https://docs.openstack.org/api-ref/database/#update-resources-quota-for-a-specific-project thus the support is incomplete.

**Comment by mandre** [comment]
Indeed, this was automatically closed as completed because the related PR included the magic `Fixes` keyword. I'll re-open because we still don't have PUT.
